client/apps.go

```diff
@@ -70,3 +76,62 @@ func (client *Client) Apps(names []string, opts AppOptions) ([]*AppInfo, error)
 
 	return appInfos, err
 }
+
+// LogOptions represent the options of the Logs call.
+type LogOptions struct {
+	N      int  // The maximum number of log lines to retrieve initially. If <0, no limit.
```

Comment:
Hmm. `journalctl` does't really give you a way to get the first N lines other than getting them all. If someone were to implement a syslog backend they'd have to look at old `.gz` files to figure out what "first" means.

When do you want to see the first lines and not "everything"?

N being _actually_ 0 is useful together with `follow`, fwiw.